src/main/java/com/redhat/cloud/common/clowder/configsource/BrokerConfig.java

```diff
@@ -10,4 +10,5 @@ public class BrokerConfig {
     public String cacert;
     public String authtype;
     public SaslConfig sasl;
+    public String securityProtocol;
```

Comment:
`securityProtocol` is already present in `SaslConfig`. It doesn't look normal to have it here as well.

Reply:
Seems also weird to me, but according [clowder-kafka documentation](https://redhatinsights.github.io/clowder/clowder/dev/providers/kafka.html) securityProtocol have to be at this place and sometimes it could also appears on subsections like sasl.